LibNFS.xs

```diff
@@ -1232,8 +1229,8 @@ set (SV* self_sv, ...)
 
                 continue;
             }
-#ifdef NLNFS_NFS_SET_READDIR_MAX_BUFFER_SIZE
 
+#ifdef NLNFS_NFS_SET_READDIR_MAX_BUFFER_SIZE
```

Comment:
Yeah I tried that... didn't work well because the Perl XS preprocessor (forgot the name) thought this is a new function.

Reply:
xsubpp